test/tensor/test_qtensor.py

```diff
@@ -8,6 +8,13 @@
 from quanto import QTensor, absmax_scale
 
 
+def test_qtensor_move(device):
+    input_shape = (2, 4, 8)
+    qa = random_qtensor(input_shape, dtype=torch.float32)
+    qa = qa.to(device)
+    assert qa._data.device.type == device.type
+    assert qa._scale.device.type == device.type
```

Comment:
This works even before this PR. This is why you never had this issue.

Reply:
There is already a test for this here: https://github.com/huggingface/quanto/blob/6302171b7569a3fd86f31e1731d01d390d1eb557/test/tensor/ops/test_quantized_dispatch.py#L8